pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -291,13 +291,13 @@ func (s *Server) initInprocessAnalysisController(args *PilotArgs) error {
 	s.addStartFunc("analysis controller", func(stop <-chan struct{}) error {
 		go leaderelection.
 			NewLeaderElection(args.Namespace, args.PodName, leaderelection.AnalyzeController, args.Revision, s.kubeClient).
-			AddRunFunction(func(stop <-chan struct{}) {
-				cont, err := incluster.NewController(stop, s.RWConfigStore,
+			AddRunFunction(func(leaderStop <-chan struct{}) {
```

Comment:
this is just syntax change, right? No behavior change?

Reply:
yes, here no behavior change.